api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -402,6 +402,10 @@
      */
     Attribute<Boolean> BUFFER_SPOOLING = new Attribute<Boolean>(true);
     /**
+     * Whether to apply back pressure and stall the upstream operator when the downstream operator is slow.
+     */
+    Attribute<Boolean> BUFFER_BACK_PRESSURE = new Attribute<Boolean>(true);
```

Comment:
It should be OK to have per port attribute. In a case back pressure is enabled only for a subset of an operator output ports, the operator will proceed at the speed of the slowest port that in this case will most likely be the port where the back pressure is enabled. It is similar to the case where an operator may have two ports/streams and one stream is container local with a small queue size. Assuming that the downstream operators (both local and remote) process tuples at the same rate, the operator will be blocked emitting to inline stream (due to small size).